### DISCUSSION TITLE
AAChartKit and Highcharts License Models?

### DISCUSSION THREAD

**Discussion by SvenMuc** [AAChartModel/AAChartKit#1329] [Q&A]
According your readme AAChartKit is based on the Highcharts JS library.

AAChartKit is licensed according the MIT license whereas Highcharts only offers commercial licenses.

https://shop.highcharts.com/?_ga=2.219808154.2087670925.1652644062-753489529.1652131962

Can you elaborate which source code dependencies exist and which license models are used? 

Thanks!

**Comment by xxriyariya-collab** [comment]
Hii

**Comment by xxriyariya-collab** [comment]
Kaya bata sakte hai kaise karate hai
